Src/GoogleApis.Auth.DotNet4/OAuth2/GoogleCredential.cs

```diff
@@ -36,10 +32,24 @@ namespace Google.Apis.Auth.OAuth2
     /// See <see cref="GetApplicationDefaultAsync"/> for the credential retrieval logic.
     /// </para>
     /// </summary>
-    public abstract class GoogleCredential : IConfigurableHttpClientInitializer, ITokenAccess
+    public class GoogleCredential : IConfigurableHttpClientInitializer, ITokenAccess
     {
         private static DefaultCredentialProvider defaultCredentialProvider = new DefaultCredentialProvider();
 
+        private readonly object credential;
+        private readonly ITokenAccess tokenAccess;
+        private readonly IConfigurableHttpClientInitializer clientInitializer;
+
+        /// <summary>
+        /// Creates a new <c>GoogleCredential</c>.
```

Comment:
You can leave it in one line, (short comment).

I'll add a todo that credential should be some interface and not just object,